backend/app/crud/users_table.py

```diff
@@ -15,6 +15,15 @@ def __init__(self, db: Database):
         self.table_id = optional_table_id
         self.db = db
 
+    def parse_user_data(self, user: dict) -> dict:
+      user_data = user.get('fields', {})
+      user_data['id'] = user.get('id', '')
+
+      user_data.setdefault('linkedin', '')
+      user_data.setdefault('description', '')
```

Comment:
I think maybe we can add this values as default in the `User` model. That way there's no need to create an extra method. What do you think?